docs/technical-documentation/migration-islandora-workbench.md

```diff
@@ -1,64 +1,67 @@
 ## Islandora Workbench
 
+Islandora Workbench provides a command-line solution to create, update, and delete Islandora content from CSV data. The Islandora Workbench repository can be found on [Github](https://github.com/mjordan/islandora_workbench). Full details and documentation for Islandora Workbench are available [here](https://mjordan.github.io/islandora_workbench_docs/).
 
-### Repository
-
-<https://github.com/mjordan/islandora_workbench>
 
 ### Overview
 
-More tailored for end users with less technical knowledge or limited server access.
+Islandora Workbench is tailored towards end users with less technical knowledge or limited server access. 
 
-- Uses Islandora’s REST API
-- Runs on your computer
-- “CSVs and a pile of scans”
-- Cross Platform - Python
+Workbench provides an alternative to Drupal's Migrate framework, as it does not need to be run on the Drupal server. Islandora Workbench uses Islandora's REST API and offers cross-platform support (Windows, Mac, Linux) to run on your computer, using your provided CSVs and content.
```

Comment:
Nitpick, I would say "files" instead of "content" as content is an umbrella term for _both_ the meatadata in the CSV and the files.

Reply:
Looks good/accurate to me.